entc/gen/func.go

```diff
@@ -127,18 +124,31 @@ func pascal(s string) string {
 	return strings.Join(words, "")
 }
 
-// camel converts the given column name into a camelCase.
+// pascal converts the given name into a PascalCase.
```

Comment:
I've updated this comment to reflect (as per my limited understanding) that this function is also used for more than column name conversion, such as enum values.